File: pset1/cash/cash.c
#include <cs50.h>
#include <stdio.h>

int main(void){


    float cents;
    int counter=0;
    //int counter2=0;
    do{
        cents = get_float("Change owed: ");
    }

    while(cents<0.0); //while negative value keep repeating

    cents = cents*100; //gives value in cents

    int change = (int)cents;
    while (change>0){


        if(change>24){
            change = change-25;
            counter++;
        }
        else if (change>9){
            change=change-10;
            counter++;
        }
        else if(change>4){
            change=change-5;
            counter++;
        }
        else if(change>0){
            change=change-1;
            counter++;
        }
        else if(change==0){
            break;
        }


    }

    printf("%d\n", counter);
}

File: pset6/cash/cash.py
from cs50 import get_float

cents = get_float("Change owed: ")
while cents<0:
    cents = get_float("Change owed: ")
cents = cents*100
counter =0

while cents>0:
    counter  = counter +1
    if cents>24:
        cents = cents-25
    elif cents>9:
        cents = cents-10
    elif cents>4:
        cents = cents-5
    elif cents>0:
        cents = cents-1
print(counter)